src/TypeScript.UnitTests/Analyzer/JavaScriptAnalyzerTests.cs

```diff
@@ -93,11 +94,9 @@ public async Task ExecuteAnalysis_AlwaysCollectsTelemetry()
         [TestMethod]
         public async Task ExecuteAnalysis_AlwaysStartsEslintBridgeServer()
         {
-            var serverProcess = SetupServerProcess(123);
-            var client = SetupEslintBridgeClient(null);
-            var clientFactory = SetupEslintBridgeClientFactory(123, client.Object);
+            var serverProcess = SetupServerProcess();
```

Comment:
I changed the helper method and it simplified the test, we don't really care about the created client in this test.